### PR TITLE
Rsc7b

### PR DESCRIPTION
**Pull request by VOstopolets**
http://docs.ably.io/client-lib-development-guide/features/#RSC7b

### CODE REVIEW

**Comment by paddybyers** [comment]
Why did you decide to change the whitespace? It's impossible to review your changes in this PR. Please stick to the coding style of the existing code.

**Comment by mattheworiordan** [comment]
>  It's impossible to review your changes in this PR. Please stick to the coding style of the existing code.

Agreed, please resubmit this PR where we can review actual changes.

**Comment by VOstopolets** [comment]
Sorry, I didn't see that whitespaces changed automatically by default. I'll resubmit PR.

**Comment by mattheworiordan** [comment]
> Sorry, I didn't see that whitespaces changed automatically by default. I'll resubmit PR.

Do you not review your commits before committing them?

File: lib/src/main/java/io/ably/lib/http/Http.java [hunk 1]
```diff
@@ -37,769 +37,717 @@
...
-                    ErrorResponse errorResponse = ErrorResponse.fromJSON(bodyText);
-                    if (errorResponse != null) {
-                        error = errorResponse.error;
-                    }
-                } catch (JsonParseException jse) {
-                    /* error pages aren't necessarily going to satisfy our Accept criteria ... */
-                    System.err.println("Error message in unexpected format: " + bodyText);
-                }
-            }
+			if(requestBody != null) {
```
Comment (mattheworiordan) [inline lib/src/main/java/io/ably/lib/http/Http.java]: This looks like a repeat of what happened before, you're committing white space changes for no reason?

**Comment by mattheworiordan** [comment]
@VOstopolets 347b80a looks like more of the same, the commit title is "generate BuildConfig for storing version lib;" yet most of the changes are unrelated and white space?

**Comment by mattheworiordan** [comment]
@VOstopolets please squash this back into the original commits.  This PR introduces white space and then removes it in f99c469, instead just don't introduce unnecessary white space in the original commits so please squash back down into meaningful commits and we'll review the PR

**Comment by VOstopolets** [comment]
Would you like to remove whitespace from all previous commits?

**Comment by mattheworiordan** [comment]
> Would you like to remove whitespace from all previous commits?

Yes, that is what I said in "don't introduce unnecessary white space in the original commits so please squash back down into meaningful commits and we'll review the PR"

**Pull request closed without merging**